src/engraving/rendering/dev/chordlayout.cpp

```diff
@@ -3426,6 +3426,9 @@ void ChordLayout::layoutNote2(Note* item, LayoutContext& ctx)
     }
     int dots = item->chord()->dots();
     if (dots && !item->dots().empty()) {
+        if (item->fixed() && !item->visible()) {
+            item->setDotsHidden(true);
```

Comment:
So `item->fixed()` is only true when rhythmic slashes are toggled.  Notes of chords are all placed on the same line, and all but one note made invisible.  However, even though the dots aren't visible, they show up grayed out.  They also get laid out properly, which we would usually want to do, but for this specific circumstance we want to hide the invisible dots behind the visible one.  `setDotsHidden` prevents the dots being moved to avoid colliding with each other.

Reply:
I believe that `item->fixed()` was not originally related to rhythmic slashes; in MuseScore 3, there was also a UI to set this property manually for other notes. In MuseScore 4, the UI for this was simply removed. 
Anyway, I guess there is not really a better way to detect rhythmic slashes... at least not until we give Chords a dedicated property for that.

Reply:
For the recording, the "fixed" property *was* specifically added for the sake of slash notation.  But there is also a utility function Chord::slash() that checks for "fixed" so you don't have to bake that into code elsewhere.  The idea was eventually if slash became something a little more native, we would only have to change the definition of that utility function.

We chose to expose "fixed" in the Inspector because it turned out to have a fair number of *other* uses as well, plus it allows you to specify which line you want your slashes on.  It's definitely a shame it's no longer exposed and I'd still argue in favor of its return.  Meanwhile, I did create a plugin to allow it to be set explicitly where needed.

Reply:
Thanks @MarcSabatella!
@miiizen In that case I'd suggest to replace `item->fixed()` with `item->chord()->slash()`; the effect will be the same, but at least the meaning of the added code will be a lot clearer for the future, I think.